amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-upgrade-migrate.adoc

```diff
@@ -33,9 +33,29 @@ Before you perform the upgrade, create a backup of your files, data, and configu
 
 Follow these steps to perform the upgrade to version 5.5.0:
```

Comment:
We are in 5.7 now so we need to make it 5.x in the title and add the upgrade paths in a table. I assume it is  4.x to 5.x, but we should verify this with Adam Lipka.

Reply:
Hmm. I wonder why I thought it wasn't generic. I would put 5.5.x for the supported upgrade path and then verify this with the devs. I actually think we should be adding a small section for each minor release, but we don't have the resources to do that right now.